File: CartDB.Parser/Models/CartridgeChipModel.cs
namespace CartDB.Parser.Models
{
    public class CartridgeChipModel
    {
        public int Id { get; set; }
        public int CartId { get; set; }
        public string PartNumber { get; set; }
        public string ManufacturerName { get; set; }
        public string ManufacturerLogo { get; set; }
        public string Designation { get; set; }
        public string Type { get; set; }
        public string Package { get; set; }
    }
}

File: CartScraper/cartridgechipmodel.py
class CartridgeChip:
    def __init__(self):
        self.id = None
        self.cartid = None
        self.partnumber = None
        self.manufacturer = None
        self.designation = None
        self.type = None
        self.package = None

    def to_dict(self):
        return {
            "id": self.id,
            "cartid": self.cartid,
            "partnumber": self.partnumber,
            "manufacturer": self.manufacturer,
            "designation": self.designation,
            "type": self.type,
            "package": self.package,
        }
